src/components/Counter/index.tsx

```diff
@@ -1,25 +1,94 @@
 import React from "react";
-import { CounterStyle } from "./styles";
+import Button from "../Button";
+import { CounterStyle, ButtonContainer } from "./styles";
 
 class Counter extends React.Component {
   state = {
-    minutes: 25,
-    seconds: 0,
+    minutes: 0,
+    seconds: 5,
+    isBreakOver: true,
   };
+  // usei esses valores (5 segundos) so pra enxergar mais rapido a mudança. task eh 5 segundos e pausa eh 4 segundos
 
+  Countdown = () => {
+    const counterLogic = () => {
```

Comment:
[suggestion] It's way better now that you cleaned it up, great job! However, we can clean this logic up a bit more still. What do you think of this:
```suggestion
    const counterLogic = () => {
      if (this.state.seconds === 0 && this.state.minutes === 0) {
        clearInterval(interval);
        this.setState({
          minutes: 0,
          seconds: this.state.isBreakOver ? 4 : 5,
          isBreakOver: !this.state.isBreakOver,
        });
        return 
      }
      if (this.state.seconds === 0 && this.state.minutes > 0) {
        this.setState({
          minutes: this.state.minutes - 1,
          seconds: 59,
        });
      } else {
        this.setState({
          minutesTask: this.state.minutes,
          seconds: this.state.seconds - 1,
        });
      }
    };
    const interval = setInterval(counterLogic, 1000);
  };
```

This removes the nested ifs and also removes some of the unneeded checks. Let me know if you need any help understanding what's going on here.